### DISCUSSION TITLE
Update problem

### DISCUSSION THREAD

**Discussion by 0xhunster** [projectdiscovery/nuclei#1461] [Q&A] [answered]
When I type the command ```go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest``` and I see end of time showing below line error. And not update the script.

```go install github.com/projectdiscovery/nuclei/v2/cmd/nuclei: copying /tmp/go-build2984596365/b001/exe/a.out: open /home/lol/go/bin/nuclei: text file busy```

**Comment by ehsandeep** [comment, marked as answer]
Are you running this from bash script or something else? this looks not related to nuclei and is specific to your setup.
